EMS/core-bundle/translations/emsco-core+intl-icu.en.yml

```diff
@@ -99,6 +99,9 @@ core:
                 other   {There are # indexes}
             }
     version: 'Version {core} with Symfony {symfony}'
+dashboard:
```

Comment:
I do not like the dashboard entry at root level.

For me it needs to be in log.error or core.dashboard ?